synapse/models/material.py

```diff
@@ -3,6 +3,30 @@
 '''
 import synapse.lib.module as s_module
 
+massunits = {
+    'µg': '0.000001',
+    'microgram': '0.000001',
+    'micrograms': '0.000001',
+
+    'mg': '0.001',
+    'milligram': '0.001',
+    'milligrams': '0.001',
+
+    'g': '1',
+    'grams': '1',
+
+    'kg': '1000',
+    'kilogram': '1000',
+    'kilograms': '1000',
```

Comment:
We technically have a need to model "weight" for transport:sea:vessels, which (based on 5 minutes of Internet research) is measured indirectly as "displacement", typically in tonnes / metric tons ( 1 metric ton = 1000 kilograms ). Would we want to add this as a recognized unit?

Reply:
I left this alone for now because displacement is potentially a volume being measured by implicit mass of sea water. I'd like to put this off for a bit :D